README.md

```diff
@@ -155,6 +158,7 @@ Currently the following service providers are supported:
 ### Switzerland
 
 - [A-Region.ch](./doc/source/a_region_ch.md)
+- [Infeo.at](./doc/source/infeo_at.md)
```

Comment:
Does infeo.at offer its services also in Germany and Switzerland? It looks a little bit strange if we have a `.at` domain in the Germany and Switzerland section. Therefore I would rather remove them here and just add a link once a concrete service provider is using their services - as you did with bogenschuetz-entsorgung.de

Reply:
yes they absolutely do: Bogenschütz is the only customer in Germany. All their other customers are in Austria and Switzerland, you can see them here: https://infeo.at/index.php/referenzen/

But okay, I will remove the global ones. My extension should work for them aswell but we can then if one wants to use another customer of infeo.at test everything out and put it officially online.